lib/rucio/core/message.py

```diff
@@ -19,17 +19,14 @@
 from sqlalchemy import or_, delete, update
 from sqlalchemy.exc import IntegrityError
 
-from dogpile.cache.api import NO_VALUE
-
-from rucio.common.cache import make_region_memcached
 from rucio.common.config import config_get
 from rucio.common.exception import InvalidObject, RucioException
 from rucio.common.utils import APIEncoder
 from rucio.db.sqla import filter_thread_work
 from rucio.db.sqla.models import Message, MessageHistory
 from rucio.db.sqla.session import transactional_session
 
-REGION = make_region_memcached(expiration_time=900)
+SUPPORTED_SERVICES = ['influx', 'elastic', 'email', 'activemq']
```

Comment:
This is a well-defined list, let's use an Enum here. This has several advantages:
- We can test it.
- We don't have to pass strings around, it acts as a "Type"
- We can implement custom function such as `from_string`, which automatically test for corrupted inputs.

Reply:
Enum was implemented. Can you please have a look ?

Reply:
:+1: